compiler-core/templates/docs-js/index.js

```diff
@@ -25,7 +25,7 @@ window.Gleam = (function () {
     return Array.from(arguments).reduce(
       (acc, name) =>
         `${acc}
-        <svg class="icon icon-${name}"><use xlink:href="#icon-${name}"></use></svg>`,
+        <svg alt="icon-${name}" class="icon icon-${name}"><use xlink:href="#icon-${name}"></use></svg>`,
```

Comment:
`alt` isn't a valid tag on SVGs, and I don't think this value will help people using screen readers

Reply:
Right
https://stackoverflow.com/questions/4697100/accessibility-recommended-alt-text-convention-for-svg-and-mathml
swap for title?